src/fragments/lib/push-notifications/react-native/getting_started/30_existing_resources.mdx

```diff
@@ -0,0 +1,17 @@
+Amplify reads configuration information at runtime from a file called `aws-exports.js`. If you used the Amplify CLI to create backend resources, this file was created for you. If it does not already exist, create the file and add it to the `src` folder in your React Native project.
```

Comment:
This was another instance of me just aligning (read: copy/pasting) from the work from the Swift/Android teams. @abdallahshaban557 @tannerabread I'm happy to go with what ever we consider best - but should we also update for the Swift/Android PN docs?

Reply:
I think for JS - we should always recommend configuring the library in code, rather than using the aws-exports (good catch!). Especially since we'll be adding types there to help with the process. Let us do that way for this guide as well.